src/pytorch_lightning/utilities/deepspeed_model_summary.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```

Comment:
for organisation, it would be nice if it was called model_summary_deepspeed or if it was grouped under a folder model_summary